android/build.gradle

```diff
@@ -63,7 +63,9 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.2.0'
 
     api("com.revenuecat.purchases:purchases:$purchases_version")
```

Comment:
should this one actually be `implementation`?

Reply:
i had the same question :) this is so hybrids can just add `purchases-hybrid-common` and don't also have to add purchases.